Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic.yml

```diff
@@ -1141,6 +1142,178 @@ tasks:
+    id: "84"
+    taskid: 3a2b054d-f7b4-4576-8d76-2bc32cf07f96
+    type: condition
+    task:
+      id: 3a2b054d-f7b4-4576-8d76-2bc32cf07f96
+      version: -1
+      name: Is Cortex Core integration enabled and user name is defined?
+      description: Checks if Cortex XDR integration is enabled and the user name is defined.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "28"
+      "yes":
+      - "85"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Cortex Core - IR
+                    ignorecase: true
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.XDRUsernameField
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.Username
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1490,
+          "y": 550
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "85":
+    id: "85"
+    taskid: 0adef6b2-c579-4948-8bbc-73f122ed38c6
+    type: title
+    task:
+      id: 0adef6b2-c579-4948-8bbc-73f122ed38c6
+      version: -1
+      name: Cortex Core
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "86"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1490,
+          "y": 800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "86":
+    id: "86"
+    taskid: 9605cc83-3071-40c2-82b7-db1775a8863f
+    type: playbook
+    task:
+      id: 9605cc83-3071-40c2-82b7-db1775a8863f
+      version: -1
+      name: Get entity alerts by MITRE tactics
+      playbookName: Get entity alerts by MITRE tactics
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      EntityID:
+        complex:
+          root: inputs.Username
```

Comment:
The playbook failed when the input `username` contains fqdn (domain\usename)
maybe we should filter the user with `\` and use a transformer to remove the `domain\`.

Reply:
Done